docs/api.md

```diff
@@ -345,6 +348,30 @@ console.log(await page.evaluate('location.href'));
 
 > **NOTE** Use [`page.waitForLoadState([state[, options]])`](#pagewaitforloadstatestate-options) to wait until the page gets to a particular state (you should not need it in most cases).
 
+#### browserContext._disableScreencast()
```

Comment:
Let's land them as commented out for now. `<!-- -->`

Reply:
Removed these. Added options.